src/vs/workbench/browser/parts/quickopen/quickOpenController.ts

```diff
@@ -479,6 +483,21 @@ export class QuickOpenController extends WorkbenchComponent implements IQuickOpe
 
 		this.previousValue = prefix;
 
+		// Track active editor before navigation
```

Comment:
@wprater what about the picker? I think you need to extract this into a reusable method so that you can also use it when the picker opens.